src/gameplay_v2/roles/components/assassinate.ts

```diff
@@ -0,0 +1,8 @@
+import Component from './component';
+
+export class assassinateC extends Component {
+  name = 'Assassinate';
+  //player which can do assassination
+  player: string = null;
```

Comment:
This shouldn't be here. Should be in `data` below!

Reply:
Fixed as well :)